pkg/cmd/update/rules_engine/fixtures/response.json

```
@@ -0,0 +1,26 @@
+{
+    "results": {
+        "id": 1234,
+        "name": "UpdatedRule",
+        "phase": "request",
+        "behaviors": [
+          {
+            "name": "run_function",
+            "target": "9045"
+          }
+        ],
+        "criteria": [
+          [
+            {
+              "variable": "${uri}",
+              "operator": "starts_with",
+              "conditional": "if",
+              "input_value": "/"
+            }
+          ]
+        ],
+      "is_active": true,
+      "order": 1
+    },
+    "schema_version": 3
+  }
```

Maybe need to add a new line here too...
```suggestion
  }
  
```